src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -656,6 +656,10 @@ export class HaConfigDevicePage extends LitElement {
                     integrations.length
                       ? html`
                           <img
+                            alt=${domainToName(
```

Comment:
Should this be `integrations[0].title`?  I'm not sure of the difference.  @bramkragten can you assist?

Reply:
Title is what the user named the config entry, `domainToName` return the integration name (translated if needed)

Reply:
This is correct as is then.  Thanks for the clarification.